### ISSUE TITLE
add official and fan buttonmen sets from old site

### ISSUE THREAD

**Issue by ManvilleFog** [buttonmen-dev/buttonmen#280]
This is part of #229
Since there are always new buttons, sets, and skills to be found, the buttonmen list is always going to be incomplete.  The only way to allow issues to be closed on this matter is to split it into smaller steps that can be considered complete, until we get to a point where we're waiting on new buttons and sets to be found and added.  

The first part is to remake what was available on the old site. My plan was to accomplish this in three stages: the official buttonmen and sets, followed by fan and vanity sets (7 deadly sins, Victorian Horror, etc.), and finally adding fanatics.  All but one or two of the official sets are complete, so I'm skipping that as a seperate issue.  So, there's this, then fanatics, and that will complete the buttonmen sets as far as the old site was concerned.  

One thing - I'm not satisfied with some of the sets - particularly the rare / promo sets, and hodgepodge.  I don't know the origin of most of hodgepodge, but some should have been part of promo sets.  And rather than grabbing all the promos from a specific year and throwing them into one category, my preference is to separate them into the groups, cons, or companies that they're meant to promote.
